pkgs/development/python-modules/grep-ast/default.nix

```
@@ -0,0 +1,41 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+
+  pathspec,
+  pytestCheckHook,
+  setuptools,
+  tree-sitter-languages,
+}:
+
+buildPythonPackage rec {
+  pname = "grep-ast";
+  version = "0.3.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "grep_ast";
+    hash = "sha256-QriIfVcwHcVWNDaPjVSenEnJE9r7TRnJtUw922BPzPQ=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pathspec
+    tree-sitter-languages
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "grep_ast" ];
+
+  meta = {
+    homepage = "https://github.com/paul-gauthier/grep-ast";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ greg ];
+    description = "A python interface to the ast-grep tool";
```

*`meta.description` must: [...] not start with the definite or an indefinite article.*
> https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

```suggestion
    description = "Python interface to the ast-grep tool";
```

---

Done